### DISCUSSION TITLE
Case of Table names generated by Prisma ORM

### DISCUSSION THREAD

**Discussion by tippfelher** [prisma/prisma#19988] [Q&A]
Hey all, I have a question which I already asked on SO ([here](https://stackoverflow.com/questions/76486183/case-of-table-names-generated-by-prisma-orm)) which got no attention, so I ask again (here).

I am using prisma ^4.12.0 in my application and use this command

> pnpm prisma migrate dev

to generate tables in my empty dev database.

This works fine, but it has different behaviour on Windows and Linux:

On Windows this command creates a table with name

> mysupercooltable

On Linux this command creates a table with name

> MySuperCoolTable

Is there a setting to enforce the same behaviour on both systems?

**Comment by nurul3101** [comment]
Hello @tippfelher 👋 

Could you share the schema file that you are using to create the tables?
Also, which database are you using?

**Comment by janpio** [comment]
This is most probably down to a configuration difference of your databases. If you generated the migrations with `migrate dev`, then the generated SQL should be optimal for cross platform compatibility - but if your database is configured to ignore that anyway, then there is nothing Prisma can do.

Can you share the generated SQL tha leads to `mysupercooltable` and `MySuperCoolTable` at the same time?

**Reply by nurul3101** [reply]
@tippfelher Checking to see if you had a chance to look at Jan's response.

It would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 
Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏
